{"uid":"30dbb362a81fc419","name":"setUp","fullName":"Base.BaseTest.setUp","historyId":"e3babf17364abe3d446637cb7829389a","time":{"start":1755754672853,"stop":1755754672856,"duration":3},"status":"broken","statusMessage":"Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. \nHost info: host: 'CCLB140-RITESHS', ip: '192.168.1.26'\nBuild info: version: '4.34.0', revision: '707dcb4246*'\nSystem info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '22.0.2'\nDriver info: io.appium.java_client.android.AndroidDriver\nCommand: [null, newSession {capabilities=[Capabilities {appium:automationName: UiAutomator2, appium:deviceName: emulator-5554, appium:plugin:images: true, browserName: Chrome, platformName: ANDROID}]}]\nCapabilities {appium:automationName: UiAutomator2, appium:deviceName: emulator-5554, appium:plugin:images: true, browserName: Chrome, platformName: ANDROID}","statusTrace":"org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. \nHost info: host: 'CCLB140-RITESHS', ip: '192.168.1.26'\nBuild info: version: '4.34.0', revision: '707dcb4246*'\nSystem info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '22.0.2'\nDriver info: io.appium.java_client.android.AndroidDriver\nCommand: [null, newSession {capabilities=[Capabilities {appium:automationName: UiAutomator2, appium:deviceName: emulator-5554, appium:plugin:images: true, browserName: Chrome, platformName: ANDROID}]}]\nCapabilities {appium:automationName: UiAutomator2, appium:deviceName: emulator-5554, appium:plugin:images: true, browserName: Chrome, platformName: ANDROID}\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)\r\n\tat io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:173)\r\n\tat io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:101)\r\n\tat io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:113)\r\n\tat io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:109)\r\n\tat drivers.DriverManager.initializeDriver(DriverManager.java:65)\r\n\tat Base.BaseTest.setUp(BaseTest.java:35)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)\r\n\tat org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:657)\r\n\tat org.testng.TestRunner.beforeRun(TestRunner.java:646)\r\n\tat org.testng.TestRunner.run(TestRunner.java:612)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)\r\n\tat org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\r\n\tat java.base/java.lang.Thread.run(Thread.java:1570)\r\nCaused by: java.io.UncheckedIOException: java.net.ConnectException\r\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)\r\n\tat io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)\r\n\tat io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)\r\n\tat org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)\r\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)\r\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)\r\n\t... 4 more\r\nCaused by: java.net.ConnectException\r\n\tat java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:956)\r\n\tat java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:133)\r\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)\r\n\t... 9 more\r\nCaused by: java.net.ConnectException\r\n\tat java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1065)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:227)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:280)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$2(PlainHttpConnection.java:238)\r\n\tat java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)\r\n\tat java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)\r\n\tat java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)\r\n\tat java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)\r\n\t... 3 more\r\nCaused by: java.nio.channels.ClosedChannelException\r\n\tat java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:204)\r\n\tat java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:873)\r\n\tat java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:961)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:210)\r\n\tat java.base/java.security.AccessController.doPrivileged(AccessController.java:571)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:212)\r\n\t... 9 more\r\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[{"name":"setUp","time":{"start":1755754672160,"stop":1755754672818,"duration":658},"description":"","status":"broken","statusMessage":"Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. \nHost info: host: 'CCLB140-RITESHS', ip: '192.168.1.26'\nBuild info: version: '4.34.0', revision: '707dcb4246*'\nSystem info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '22.0.2'\nDriver info: io.appium.java_client.android.AndroidDriver\nCommand: [null, newSession {capabilities=[Capabilities {appium:automationName: UiAutomator2, appium:deviceName: emulator-5554, appium:plugin:images: true, browserName: Chrome, platformName: ANDROID}]}]\nCapabilities {appium:automationName: UiAutomator2, appium:deviceName: emulator-5554, appium:plugin:images: true, browserName: Chrome, platformName: ANDROID}","statusTrace":"org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. \nHost info: host: 'CCLB140-RITESHS', ip: '192.168.1.26'\nBuild info: version: '4.34.0', revision: '707dcb4246*'\nSystem info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '22.0.2'\nDriver info: io.appium.java_client.android.AndroidDriver\nCommand: [null, newSession {capabilities=[Capabilities {appium:automationName: UiAutomator2, appium:deviceName: emulator-5554, appium:plugin:images: true, browserName: Chrome, platformName: ANDROID}]}]\nCapabilities {appium:automationName: UiAutomator2, appium:deviceName: emulator-5554, appium:plugin:images: true, browserName: Chrome, platformName: ANDROID}\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)\r\n\tat io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:173)\r\n\tat io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:101)\r\n\tat io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:113)\r\n\tat io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:109)\r\n\tat drivers.DriverManager.initializeDriver(DriverManager.java:65)\r\n\tat Base.BaseTest.setUp(BaseTest.java:35)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)\r\n\tat org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:657)\r\n\tat org.testng.TestRunner.beforeRun(TestRunner.java:646)\r\n\tat org.testng.TestRunner.run(TestRunner.java:612)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)\r\n\tat org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\r\n\tat java.base/java.lang.Thread.run(Thread.java:1570)\r\nCaused by: java.io.UncheckedIOException: java.net.ConnectException\r\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)\r\n\tat io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)\r\n\tat io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)\r\n\tat org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)\r\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)\r\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)\r\n\t... 4 more\r\nCaused by: java.net.ConnectException\r\n\tat java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:956)\r\n\tat java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:133)\r\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)\r\n\t... 9 more\r\nCaused by: java.net.ConnectException\r\n\tat java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1065)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:227)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:280)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$2(PlainHttpConnection.java:238)\r\n\tat java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)\r\n\tat java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)\r\n\tat java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)\r\n\tat java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)\r\n\t... 3 more\r\nCaused by: java.nio.channels.ClosedChannelException\r\n\tat java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:204)\r\n\tat java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:873)\r\n\tat java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:961)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:210)\r\n\tat java.base/java.security.AccessController.doPrivileged(AccessController.java:571)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:212)\r\n\t... 9 more\r\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"stepsCount":0,"attachmentStep":false},{"name":"setUpPage","time":{"start":1755754672953,"stop":1755754672953,"duration":0},"description":"","status":"broken","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"stepsCount":0,"attachmentStep":false}],"afterStages":[{"name":"tearDown","time":{"start":1755754672983,"stop":1755754672983,"duration":0},"description":"","status":"broken","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"stepsCount":0,"attachmentStep":false}],"labels":[{"name":"package","value":"Test.WorkingWithBrowserTest"},{"name":"testClass","value":"Test.WorkingWithBrowserTest"},{"name":"testMethod","value":"setUp"},{"name":"parentSuite","value":"AppiumTestSuite"},{"name":"suite","value":"Device1Test"},{"name":"subSuite","value":"Test.WorkingWithBrowserTest"},{"name":"host","value":"CCLB140-Riteshsingh"},{"name":"thread","value":"20364@CCLB140-Riteshsingh.TestNG-tests-1(34)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"AS_ID","value":"-1"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"deviceIndex","value":"0"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"d466cf97abe2e2af","status":"broken","statusDetails":"Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: Error getting device API level. Original error: Error executing adbExec. Original error: 'Command ''C:\\\\Users\\\\Ritesh Kumar Singh\\\\AppData\\\\Local\\\\Android\\\\Sdk\\\\platform-tools\\\\adb.exe' -P 5037 -s emulator-5556 shell getprop ro.build.version.sdk' timed out after 20000ms'. Try to increase the 20000ms adb execution timeout represented by 'adbExecTimeout' capability \nHost info: host: 'CCLB140-RITESHS', ip: '192.168.1.59'\nBuild info: version: '4.34.0', revision: '707dcb4246*'\nSystem info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '22.0.2'\nDriver info: io.appium.java_client.android.AndroidDriver\nCommand: [null, newSession {capabilities=[Capabilities {appium:automationName: UiAutomator2, appium:deviceName: emulator-5554, platformName: ANDROID}]}]\nCapabilities {appium:automationName: UiAutomator2, appium:deviceName: emulator-5554, platformName: ANDROID}","time":{"start":1755681379243,"stop":1755681379245,"duration":2}}],"categories":[{"name":"Test defects","matchedStatuses":[]}],"tags":[]},"source":"30dbb362a81fc419.json","parameterValues":["0"]}